dsl_parser/elements/imports.py

```diff
@@ -101,10 +137,17 @@ def validate(self, **kwargs):
         imports = [i.value for i in self.children()]
         imports_set = set()
         for _import in imports:
-            if _import in imports_set:
-                raise exceptions.DSLParsingFormatException(
-                    2, 'Duplicate imports')
-            imports_set.add(_import)
+            if isinstance(_import, str):
```

Comment:
If you make the conditional set import_key = _import (or import_key = _validate_import_dict(_import) when a dict), then you can write the raising/adding just once.